modular_skyrat/modules/novaya_ert/code/toolbox.dm

```diff
@@ -53,12 +53,12 @@
 /obj/item/storage/toolbox/ammo/full/bison
 	name = "ammo box (PP-95)"
 	desc = "It contains a few magazines."
-	ammo_type = /obj/item/ammo_box/magazine/multi_sprite/cfa_lynx
+	ammo_type = /obj/item/ammo_box/magazine/plastikov9mm
 	amount = 4
```

Comment:
Swap to Uzi mags; those ammo boxes are meant to be used by the NRI SMG, and were called plastikov because I was too lazy to rename it (and I mean why nobody would ever look into those files).

Reply:
This post is misinformation.